app/controllers/sessions_controller.rb

```diff
@@ -4,6 +4,7 @@ class SessionsController < ApplicationController
 
   def new
     if current_user.present?
+      current_user.update_attribute(:last_login, Time.now)
```

Comment:
For the future, use the Rails defined method [`touch`](http://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-touch)